sourcecode/test_weather_services.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from sourcecode.weather_services import WeatherMan
+
+class TestWeatherServices(unittest.TestCase):
+
+    path = '/Users/abubakarkhawaja/Documents/weatherfiles/'
+    obj = WeatherMan()
+
+    def test_get_weather_info(self):
+        file_path = self.path + 'Murree_weather_2004_Aug.txt'
```

Comment:
Instead of parsing a complete file let's make a small dummy file with test data and use it.